survey-service/src/main/java/pl/hycom/surveyservice/controller/EditSurveyController.java

```diff
@@ -21,9 +21,10 @@
     @Autowired
     SurveyRepository surveyRepository;
 
-    @RequestMapping(value = "/getSurvey/{surveyId}", method = RequestMethod.GET)
-    public ResponseEntity<Survey> getSurveyToEdit(@PathVariable String surveyId) {
+    @RequestMapping(value = "/editSurvey/{surveyId}", method = RequestMethod.GET)
```

Comment:
Ogólnie to tak, patrząc na endpointy jest lepiej.
Aczkolwiek dalej mamy czasownik w URL-u.

Można pomyśleć, czy nie możemy obsłużyć pobierania ankiety do edycji i wypełniania jednym endpointem.

W sumie, to potrzebujemy te same dane, nie prawdaż?

P.S. jak zmieniłeś w backendzie to pewno też w reacie trzeba zmienić endpointy? ;P